hdl/projects/grapefruit/grapefruit_pins.xdc

```diff
@@ -117,13 +117,18 @@ set_property -dict { PACKAGE_PIN W1 IOSTANDARD LVCMOS18 } [get_ports { fpga_spar
 set_property -dict { PACKAGE_PIN U5 IOSTANDARD LVCMOS18 } [get_ports { fpga_spare_1v8[1] }];
 set_property -dict { PACKAGE_PIN W2 IOSTANDARD LVCMOS18 } [get_ports { fpga_spare_1v8[0] }];
 set_property -dict { PACKAGE_PIN Y1 IOSTANDARD LVCMOS18 } [get_ports { espi_hpm_to_scm_reset_l }];
-set_property -dict { PACKAGE_PIN U2 IOSTANDARD LVCMOS18 } [get_ports { espi_scm_to_hpm_alert_l }];
+# espi boot wants alert to be data[1], but it's annoying to carry that rename through the desig
+# so we swapped them here
+#set_property -dict { PACKAGE_PIN U2 IOSTANDARD LVCMOS18 } [get_ports { espi_scm_to_hpm_alert_l }];
+set_property -dict { PACKAGE_PIN U4 IOSTANDARD LVCMOS18 } [get_ports { espi_scm_to_hpm_alert_l }];
 set_property -dict { PACKAGE_PIN U1 IOSTANDARD LVCMOS18 } [get_ports { espi_hpm_to_scm_cs_l }];
 set_property -dict { PACKAGE_PIN W3 IOSTANDARD LVCMOS18 } [get_ports { espi_hpm_to_scm_clk }];
 set_property -dict { PACKAGE_PIN T3 IOSTANDARD LVCMOS18 } [get_ports { espi_hpm_to_scm_dat[3] }];
 set_property -dict { PACKAGE_PIN U3 IOSTANDARD LVCMOS18 } [get_ports { espi_hpm_to_scm_dat[2] }];
-set_property -dict { PACKAGE_PIN U4 IOSTANDARD LVCMOS18 } [get_ports { espi_hpm_to_scm_dat[1] }];
+#set_property -dict { PACKAGE_PIN U4 IOSTANDARD LVCMOS18 } [get_ports { espi_hpm_to_scm_dat[1] }];
```

Comment:
Is there value in leaving this line (128) and 122 above commented out in the xdc given your comment on 120-121?

Reply:
Probably not, removed